pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -42,6 +42,11 @@ type CatalogSourceSpec struct {
 	// SourceType is the type of source
 	SourceType SourceType `json:"sourceType"`
 
+	// Priority field, if specified, is used to rank catalog sources that contain operators
```

Comment:
Having "if specified" here is redundant since the parameter is marked as `+Optional`